frontstage/views/passwords/reset_password.py

```diff
@@ -43,6 +43,9 @@ def get_reset_password(token, form_errors=None):
     except (BadSignature, BadData):
         logger.warning("Invalid token sent to frontstage password reset", token=token, exc_info=True)
         return render_template("passwords/password-expired.html", token=token)
+    except TypeError:
```

Comment:
We really shouldn't be doing this, this is catching a bug, rather than fixing it. The 404 should be raised and handled correctly